relayer-cli/src/commands/tx/packet.rs

```diff
@@ -48,6 +50,14 @@ pub struct TxPacketRecvCmd {
         help = "Identifier of the source channel"
     )]
     src_channel_id: ChannelId,
+
+    #[clap(
+        long = "packet-data-query-height",
+        required = false,
+        help_heading = "NOT REQUIRED",
```

Comment:
We should maybe make this prettier, unclear what is does exactly and if its useful in more places.

Reply:
The `help_heading` is used to specify a category for the flag, so with this one set to `NOT REQUIRED` the help output is:
```
hermes-tx-packet-recv 
Relay receive or timeout packets

USAGE:
    hermes tx packet-recv [OPTIONS] --dst-chain <DST_CHAIN_ID> --src-chain <SRC_CHAIN_ID> --src-port <SRC_PORT_ID> --src-channel <SRC_CHANNEL_ID>

OPTIONS:
    -h, --help    Print help information

REQUIRED:
        --dst-chain <DST_CHAIN_ID>        Identifier of the destination chain
        --src-chain <SRC_CHAIN_ID>        Identifier of the source chain
        --src-channel <SRC_CHANNEL_ID>    Identifier of the source channel [aliases: src-chan]
        --src-port <SRC_PORT_ID>          Identifier of the source port

NOT REQUIRED:
        --packet-data-query-height <PACKET_DATA_QUERY_HEIGHT>
            Height at which the packet data is queried
```

But the CLIs all have the optional flags under the `OPTIONS` category, so I will remove the `help_heading` in order to have the help output:
```
hermes-tx-packet-recv 
Relay receive or timeout packets

USAGE:
    hermes tx packet-recv [OPTIONS] --dst-chain <DST_CHAIN_ID> --src-chain <SRC_CHAIN_ID> --src-port <SRC_PORT_ID> --src-channel <SRC_CHANNEL_ID>

OPTIONS:
    -h, --help
            Print help information

        --packet-data-query-height <PACKET_DATA_QUERY_HEIGHT>
            Height at which the packet data is queried

REQUIRED:
        --dst-chain <DST_CHAIN_ID>        Identifier of the destination chain
        --src-chain <SRC_CHAIN_ID>        Identifier of the source chain
        --src-channel <SRC_CHANNEL_ID>    Identifier of the source channel [aliases: src-chan]
        --src-port <SRC_PORT_ID>          Identifier of the source port
```